akka-docs/rst/java/persistence.rst

```diff
@@ -385,6 +385,10 @@ Deleting messages in event sourcing based applications is typically either not u
 up until the sequence number of the data held by that snapshot can be issued to safely delete the previous events
 while still having access to the accumulated state during replays - by loading the snapshot.
 
+.. warning::
+  If you are using :ref:`persistence-query-java`, query results will be missing deleted messages in a journal.
```

Comment:
Thanks for the feedback. I modified the warning text - could you see if that looks ok?